src/utils/devhub-api-stitch.js

```diff
@@ -74,6 +74,22 @@ export const submitStudentSpotlightProject = async projectData => {
     return result;
 };
 
+export const submitCommunityChampionApplication = async (
+    communityChampionData,
+    onSuccess,
+    onFailure
+) => {
+    try {
+        const result = await callDevhubAPIStitchFunction(
```

Comment:
👏  putting this logic in here is awesome since we abstract it from the form entirely, nice!